common/wonders/wc_wonders_buildable.txt

```diff
@@ -364,3 +364,124 @@ wonder_soul_engine = {
 		}
 	}
 }
+wonder_statue_yak = {
+	picture = GFX_wonder_strip_yak_statue
+	allow_coastal = yes
+	potential = {
+		# Warcraft
+		evil_public_religion_trigger = no
+		
+		OR = {
+			culture = yaungol
+			culture = grummle
+		}
+	}
+	active = {
+		ROOT = {
+			# Warcraft
+			evil_public_religion_trigger = no
+			
+			OR = {
+				culture = yaungol
+				culture = grummle
+			}
+		}
+	}
+	flags = {
+		statue
+		yak_statue
+		inspires_greatness
+		is_destructible
+		cost_low
+		is_historical
```

Comment:
It's not historical.
```suggestion
```

Reply:
it's used to block dynamic localization being used when upgrading the wonder stage

Reply:
> it's used to block dynamic localization being used when upgrading the wonder stage

Well, it doesn't work then.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/92765627-92215480-f3a6-11ea-9969-9c57a99f5928.png)

</details>